.github/workflows/release.yaml

```diff
@@ -0,0 +1,81 @@
+---
+name: release
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  inspect:
+    runs-on: ubuntu-latest
+    container: golang:1.13
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set version
+        run: echo '::set-output name=version::${GITHUB_REF#refs/*/}'
+        id: set_version
+    outputs:
+      version: ${{ steps.set_version.outputs.version }}
+  release:
+    runs-on: ubuntu-latest
+    needs: inspect
+    if: ${{ needs.inspect.outputs.version }}
+    steps:
+      - name: Create the GitHub release.
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+        with:
+          tag_name: ${{ needs.inspect.outputs.version }}
+          release_name: ${{ needs.inspect.outputs.version }
+          draft: false
+          prerelease: false
+    outputs:
+      upload_url: ${{ steps.create_release.outputs.upload_url }}
+  build:
+    runs-on: ubuntu-latest
+    needs:
+      - inspect
+      - release
+    strategy:
+      matrix:
+        osarch:
+          - os: linux
+            arch: amd64
+          - os: linux
+            arch: arm
+          - os: darwin
+            arch: amd64
+          - os: windows
+            arch: amd64
+    steps:
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "1.13"
+      - uses: actions/checkout@v2
+      # This project does not use these,  but we need them to build the
+      # binaries.
```

Comment:
I find this comment confusing. Could you clarify?

Reply:
They are not a compile-time dependency consumed by the code being compiled, rather they are build tools.

Reply:
I clarified it a bit more.